responses/40-whats-next.md

```diff
@@ -1,22 +1,22 @@
 You are doing a great job, @{{ user.username }}! :star2: :collision: :tropical_fish:
 
-But you may be asking why we asked you to go through all of the hard work of connecting functions, files, and targets together using a yaml file. We don't blame you for wondering...
+But you may be asking why we asked you to go through all of the hard work of connecting functions, files, and targets together using a makefile. We don't blame you for wondering...
 
 ---
 
 The real power of depedency management is when something changes - that's the EUREKA! moment, but we haven't put you in a situation where it would show up. That will come further down the road on later training activities and also in the project work you will be exposed to. 
 
 In the meantime, here are a few nice tricks given you have a functional pipeline. 
 
-- [ ] run `scmake()` again. What happens? Hopefully not much. I see this: 
+- [ ] run `tar_make()` again. What happens? Hopefully not much. I see this: 
 ![make all is fresh](https://user-images.githubusercontent.com/2349007/82728169-763d5a00-9cb4-11ea-8ab2-ee75927b5c7e.png)
 Which means everthing is up to date so all targets are :OK:
 
-- [ ] now try making a change to one of your functions in your code. What happens after running `scmake()` then? 
+- [ ] now try making a change to one of your functions in your code. What happens after running `tar_make()` then? 
 
-- [ ] access the `plot_data` target by using `plot_data <- scmake('plot_data')`. (You may or may not have an R-object target named `plot_data` in your own repo at this point, so go ahead and try it with some target that you do have.) Here we are using the first argument of `scmake()`, which is `target_names`. Any vector of targets used will be built (the default is to build the `all` target in the case where no target names were specified). We can access the output of the target by assigning the result to a variable. In this example, we have called that variable `plot_data`, and it receives output from `scmake` in the form of a data.frame because that's what our example function `process_data()` creates. If you assign the result of a file target, like `file_name <- scmake(target_names = '1_fetch/out/model_RMSEs.csv')`, the result is the path to that file.
+- [ ] access the `plot_data` target by using `plot_data <- tar_read(plot_data)`. (You may or may not have an R-object target named `plot_data` in your own repo at this point, so go ahead and try it with some target that you do have.) In this example, we have passed in the unquoted target name `plot_data`, and it receives output from `tar_read` in the form of a data.frame because that's what our example function `process_data()` creates. If you assign the result of a file target, like `file_name <- tar_read(model_RMSEs_csv)`, the result is the path to the target's file.
```

Comment:
I like `tar_load()` for inspecting objects and/or debugging other functions it feeds into.

Reply:
Oh, nice! I will adjust :)

Reply:
Oh, just finished reading your full comment. I sort of agree that it may be too soon to tell them multiple ways to do something, so I would want to pick the simpler of the two, which I think is `tar_load()`. It seems like the only reason to use `tar_read()` and not `tar_load()` is if you want the object in your environment to be named differently than the target name. Is that how you see it?

Reply:
Yes, I agree with that and support introducing `tar_load()` only here.